README.md

```diff
@@ -1,10 +1,9 @@
 # phimpme-android
-<img src="/app/src/main/res/mipmap-xxxhdpi/ic_launcher.png" align="left" width="200" hspace="10" vspace="10">
+![Phimpme](/docs/_static/logo.png)
```

Comment:
With this fix: https://github.com/fossasia/yaydoc/pull/298; we don't need to care about the location, right?